ACPs/113-provable-randomness/README.md

```diff
@@ -0,0 +1,165 @@
+```text
```

Comment:
Would you mind updating the header to use the newly introduced table format?

https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/TEMPLATE.md

Reply:
done.